### DISCUSSION TITLE
Blacklist & Dispelable Status

### DISCUSSION THREAD

**Discussion by spamwax** [rgd87/Aptechka#77] [Q&A] [answered]
I have added Burst debuff to the Blacklist, however it shows up on my Dispelable icon widget! Is this intended or Dispelable ignores Blacklist?

**Comment by d87** [comment, marked as answer]
I guess not intended. Fixed it, but I can't make a release right now because curse is down, remind me later

**Comment by d87** [comment]
https://github.com/rgd87/Aptechka/tree/9.1-beta1

**Reply by spamwax** [reply]
I just tried that release and the debuf icons in dungeons don't show up anymore, like Mistveil Bite in Mist.
